os/fs/driver/mtd/smart.c

```diff
@@ -5163,7 +5164,8 @@ static int smart_journal_checkout(FAR struct smart_struct_s *dev, journal_log_t
  * Name: smart_journal_process_transaction
  *
  * Description:
- *   Check out journal entry, checkpoint to mtd header from journal data
+ *   Process the COMMIT/ERASE/RELEASE task onto MTD flash memory based on
+ *   the type of task. Proceed with readback and verification if required.
```

Comment:
instead of readback please add comment regarding validation checking of crc, if type is commit.

Reply:
Added, thanks